po/ko.po

```diff
@@ -1168,60 +1132,6 @@ msgstr ""
 "* <kbd>Ctrl + Enter</kbd>: 현재 포커스를 받은 코드 샘플 블록을 실행합니다.\n"
 "* <kbd>s</kbd>: 검색창을 활성화합니다.(mdbook 문제로 23.01.19 기준 영어로만 가능합니다.)"
 
-#: src/running-the-course/translations.md:1
```

Comment:
It looks like this got deleted by accident?

Reply:
Sorry, and thanks for pointing that out. Fixed.